### DISCUSSION TITLE
Add ability to add name or label to a Langfuse API key

### DISCUSSION THREAD

**Discussion by agibralter** [langfuse/langfuse#2642] [Ideas]
### Describe the feature or potential improvement

It would be helpful to be able to label or name different API keys (e.g. for different application environments). Right now, the only label is the date created and the public key itself which is not very user-friendly.

**Comment by clemra** [comment]
Thank you. This is a very helpful suggestion that we agree with.

**Comment by marcklingen** [comment]
Thanks again for the feedback, added this here: https://github.com/langfuse/langfuse/pull/4686
